Neos.Flow/Classes/Error/DebugExceptionHandler.php

```diff
@@ -54,7 +54,7 @@ class DebugExceptionHandler extends AbstractExceptionHandler
     /**
      * Formats and echoes the exception as XHTML.
      *
-     * @param object $exception \Exception or \Throwable
+     * @param \Throwable $exception
      * @return void
      */
     protected function echoExceptionWeb($exception)
```

Comment:
no type hint for B/C?

Reply:
I just kept the exact abstract method signature without much thinking here I guess